src/assets/pack-templates/rune-magic-spells/absorption.yaml

```
@@ -0,0 +1,23 @@
+# absorption
+name: ${{openSystem.content.runeMagic.absorption.name}}$
+type: runeMagic
+img: systems/rqg/assets/images/items/rune-magic.svg
+data:
+  runes:
+    - Earth (element) # to deprecate
+    - Darkness (element) # to deprecate
+  points: 1
+  castingRange: ranged
+  duration: temporal
+  isStackable: true
+  isOneUse: false
+  chance: 0
+  descriptionRqidLink:
+    rqid: je..absorption
+    name: ${{openSystem.content.runeMagic.absorption.name}}$
+flags:
+  rqg:
+    documentRqidFlags:
+      id: i.runeMagic.absorption
```

rqid shouldn't be camel case. Goes for the other runemagic rqids as well.
```suggestion
      id: i.rune-magic.absorption
```

---

Easy fix. I did't change it from the starter set version though so we missed it before. But for tomorrow.